src/test/ui/rfcs/rfc-2396-target_feature-safe-calls.rs

```diff
@@ -0,0 +1,19 @@
+#[target_feature(enable = "sse2")]
```

Comment:
There doesn't seem to be any run-pass tests here; please add some. In particular, it should exercise:
- Exact matches (needs / provides is the exact same set), including with one feature and with say 3 features.
- Subset matches (needs N, provides N+1, for e.g. N = 1, 2).